pip/commands/cache.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+
+import logging
+import os
+
+from pip.basecommand import Command
+from pip.status_codes import SUCCESS, ERROR
+
+
+logger = logging.getLogger(__name__)
+
+
+class CacheCommand(Command):
+    """\
```

Comment:
Better auto-dedenting behavior; IIRC if the first line of the block quote doesn't start with whitespace the rest of the docstring won't be dedented.